server/routerlicious/packages/services-shared/src/runner.ts

```diff
@@ -83,6 +83,16 @@ export async function run<T extends IResources>(
 		});
 	});
 
+	process.on('SIGINT', () => {
```

Comment:
Can we also log process.on('unhandledRejection', ...)

Reply:
@manishgargd that is already caught by the uncaughtException handler - https://nodejs.org/api/process.html#event-uncaughtexception